### ISSUE TITLE
v0.4.1 causes windows flutter application to crash

### ISSUE THREAD

**Issue by eggysoft** [mix1009/desktop_window#45]
The following code causes flutter application to crash in version 0.4.1 of the package. 

```
 if (defaultTargetPlatform == TargetPlatform.windows) {
    await DesktopWindow.setWindowSize(const Size(windowWidth, windowHeight));
  }
```

Version 0.4.0 works fine
(I performed a flutter clean, pinned the version at 0.4.0 - all ok.  Performed another fltuter clean, pinned the version at 0.4.1 and I get a "Lost connection to device." when the above future is executed.

**Comment by eggysoft** [comment]
I've done a bit more testing with a newly created app.  I created a brand new default app, added the package and included a call to getWindowSize(). A flutter clean and then run on version 0.4.0 is ok. A flutter clean and then run on version 0.4.1 crashes.  So although 0.4.1 made changes to the MacOS flow on setWindowSize(), I think (with very little knowledge on the subject) that the problem crept in as part of the publish process (not that I have any idea what that could possibly be).

**Comment by mix1009** [comment]
Strange I checkout the 0.4.0 on github, but it still crashes with the latest flutter version.

git checkout a3f0f3659aac2fbf7be8dac9040568df4110aab2 

Windows code was contributed, so i'm not too familiar with the code. But I'm trying to figure out the issue.

**Comment by eggysoft** [comment]
If I had been using 0.4.1 in my pubspec.yaml I would still get a crash on 0.4.0 until I did a flutter clean.  I wonder if it is one of the transitive dependencies that has changed that is really causing the problem.  Thank you for having a look at the issue.

**Comment by mix1009** [comment]
i've just published 0.4.2. please check if it fixes crash issue.

**Comment by eggysoft** [comment]
I've tested 0.4.2 on two different windows apps (one with no other dependencies and one with lots), both are working perfectly (in fact, I think it might work better than before - occasionally, when running in debug, the window wouldn't resize correctly after a clean build but now it is working every time) .  Thank you very much!
